### PR TITLE
make turn penalty if weight is not duration

### PR DESCRIPTION
**Pull request by MichalPP**
make turn penalty if weight is not duration

# Issue

in the bike profile, the turn penalty does not occur when weight is other than duration

## Tasklist
 - [ ] review

### CODE REVIEW

**Comment by MichalPP** [comment]
@emiltin , what do you think?

**Comment by MichalPP** [comment]
@emiltin, @danpat what do you think? it's just a post #77 profile bug

File: profiles/bicycle.lua [hunk 1]
```diff
@@ -544,5 +544,10 @@ function turn_function(turn)
       if not turn.source_restricted and turn.target_restricted then
           turn.weight = turn.weight + 3000
       end
+    if normalized_angle >= 0.0 then
```
Comment (TheMarex) [inline profiles/bicycle.lua]: What about just setting `turn.weight = turn.duration` on line 543 and removing this code? I think this was a c&p bug.

Comment (MichalPP) [inline profiles/bicycle.lua]: ok, that is a lot more simpler.

**Comment by TheMarex** [comment]
Thanks for taking a stab at this! 👍  Replaced by #4044.

**Pull request closed without merging**